### ISSUE TITLE
Basic operation?

### ISSUE THREAD

**Issue by shane806** [mdesilva/AmazonFlexUnlimited#1]
Hey, idk if you'll see this since the readme states you're no longer working on this project, but I was wondering if you could explain how basic operation of this code base would work?

I guess I would say I'm an intermediate level coder but I've never worked with internet related stuff so all the http things are a bit confusing to me. 

Thanks so much in advance if you're able to respond!

-Shane

**Comment by mdesilva** [comment]
Don't worry about the http requests. You can actually just run the app with the `singleRun.py` file from your command line. However, it seems that I left some hard coded arguments, such as warehouse locations, in the code, and it is just in generally a bit messy so I might try to clean it up this week.

**Comment by shane806** [comment]
So I actually spent some time last week familiarizing myself with everything and with a few edits concerning my particular preferences I have it where I can use either singleRun.py or service.py and its been working great for me so far.

Also, the way I've been filtering the offer results is just programmatically in the findJobs() function after all offer results have been returned from getOffers() and before calling acceptOffer().

So far, the time it takes to run the filters I have in place are fast enough to where I haven't had much of a problem with ultimately being able to get the kind of offers i'm ultimately looking for. So, by doing that I've avoided the issue of having to immediately accept any/all offers and then forfeit all the ones I'm ultimately uninterested in. 

Just thought i'd mention that in case you have an interest in revising that part of flex.py because forfeiting previously accepted offers negatively impacts what Amazon calls "standing" in the Amazon Flex program, which is basically a point based score that determines what/how many offers a driver will be sent and ultimately their ability to remain a driver.

I can submit a PR if you want to see how I'm using it. Just lm
